src/irc/IrcServer.ts

```diff
@@ -159,6 +159,10 @@ export class IrcServer {
         return this.config.ircClients.userModes || "";
     }
 
+    public getRealNameFormat() {
```

Comment:
Otherwise TypeScript may imply `:string`.
```suggestion
    public getRealNameFormat(): "mxid"|"reverse-mxid" {
```

Reply:
Fair, although realnameFormat *is* typed so Typescript does the right thing.